test/Serializer/BlogArticleNormalizerTest.php

```diff
@@ -78,6 +81,31 @@ public function canNormalizeProvider() : array
 
     /**
      * @test
+     */
+    public function is_can_serialize_and_deserialize_blog_articles()
```

Comment:
I'd change the `ApiSdk` tests to test `serialize()` rather than `normalize()` instead of this (`BlogArticleNormalizer` only (de)normalizes, not (de)serialize).